ksp_plugin/vessel.hpp

```diff
@@ -371,8 +391,9 @@ class Vessel {
   RecurringThread<PrognosticatorParameters,
                   DiscreteTrajectory<Barycentric>> prognosticator_;
 
-  std::variant<std::unique_ptr<FlightPlan>,
-               serialization::FlightPlan> flight_plan_;
+  std::vector<std::variant<not_null<std::unique_ptr<FlightPlan>>,
+                           serialization::FlightPlan>> flight_plans_;
+  int selected_flight_plan_ = -1;
```

Comment:
Could we make this an `optional`?  It would be safer, just say no to magic values.

Reply:
As discussed, this would probably not help readability.